src/main/resources/default/taglib/t/blobImage.html.pasta

```diff
@@ -19,13 +21,15 @@
         <i:if test="urlBuilder.isConversionExpected()">
             <img class="lazy-image-js @imgClass"
                  style="@imgStyle"
-                 src="@urlBuilder.getFallbackUri().orElse(sirius.biz.storage.layer2.URLBuilder.IMAGE_FALLBACK_URI)"
-                 data-src="@urlBuilder.buildImageURL()"/>
+                 src="@urlBuilder.withFallbackUri(fallbackUri).buildFallbackURL()"
+                 data-src="@urlBuilder.buildURL().orElse('')"
```

Comment:
```suggestion
                 src="@urlBuilder.createBaseURL().append(urlBuilder.getFallbackUri().orElse(sirius.biz.storage.layer2.URLBuilder.IMAGE_FALLBACK_URI))"
                 data-src="@urlBuilder.buildImageURL()"
```

Reply:
Otherwise we overwrite the existing fallback uri (if present)....

Reply:
Are you sure about @urlBuilder.buildImageURL()?

Reply:
Yep, otherwise its an empty string... :D